tests/test_iotedgemodule.py

```diff
@@ -0,0 +1,34 @@
+import json
+import logging
+import os
+import pytest
+
+root_dir = os.getcwd()
+module_name = "iotedgeFilterModule"
+file_list = ["requirements.txt", "module.json", "main.py", "Dockerfile.arm32v7",
+             "Dockerfile.amd64.debug", "Dockerfile.amd64", ".gitignore"]
+image_repository="localhost:5000/"+module_name
+
+
+@pytest.fixture(scope="module")
+def test_install_iotedgemodule(request):
+    installmodulecommand = "cookiecutter --no-input "+os.environ["BUILD_REPOSITORY_LOCALPATH"]+" module_name=" + \
```

Comment:
By depending on `os.environ["BUILD_REPOSITORY_LOCALPATH"]`, your test can only run on VSTS, but not locally. This can be painful when I want to see the test results without pushing the VSTS after making some changes. You can replace `os.environ["BUILD_REPOSITORY_LOCALPATH"]` with `root_dir`.

Reply:
Minor naming suggestion: `installmodulecommand` -> `scaffold_command`